packages/terra-form-select/src/combobox/Frame.jsx

```diff
@@ -468,6 +468,8 @@ class Frame extends React.Component {
       isAbove: false,
     });
 
+    this.input.focus();
```

Comment:
why do we need this to be set explicitly?

Reply:
Removed here https://github.com/cerner/terra-core/pull/2904/commits/2d8de879a0187eb87b5dea6d6877d871738c1ef4